source/_integrations/roborock.markdown

```diff
@@ -195,6 +195,12 @@ The vacuum entity holds the ability to control most things the vacuum can do, su
 - **Filter time left**
   - **Description**: How much time is left before Roborock recommends you replace your vacuum's air filter.
 
+- **Maintenance brush time left**
+  - **Description**: How much time is left before Roborock recommends you replace your dock's maintenance brush.
+
+- **Strainer time left**
+  - **Description**: How much time is left before Roborock recommends you replace your dock's strainer - this can refer to the water filter or the cleaning tray depending on your device.
```

Comment:
```suggestion
  - **Description**: How much time is left before Roborock recommends you replace your dock's strainer. This can refer to the water filter or the cleaning tray depending on your device.
```